src/calculateRentalCost.js

```diff
@@ -4,7 +4,18 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const perDay = 40;
+  let total = days * perDay;
+
+  if (days >= 7) {
+    total = total - 50;
+  } else if (days >= 3) {
+    total = total - 20;
+  }
```

Comment:
Remove the 'else' after the 'if' statement with a return to simplify the control flow as suggested in the checklist.